app/views/users/show.html.erb

```diff
@@ -10,8 +10,12 @@
     <li> <%= facade.user.first_name  %> <%= facade.user.last_name %> </li>
     <li> <%= facade.user.email%> </li>
   </ul>
+
   <section>
     <h1>Bookmarked Segments</h1>
+    <% current_user.videos.order(tutorial_id: :asc).order(position: :asc).each do |video| %>
```

Comment:
I agree with your comments this should be refactored into the model.

Reply:
Also this should now happen in the partial rather than the view. I'm thinking checking out a new branch and just doing the logic there will be best? Right now the merge conflict resolution (couldn't maintain the view with what we did earlier) is to just skip the test that fails as a result of this.